dss-core/src/main/java/io/github/ztkmkoo/dss/core/actor/rest/DssRestExceptionHandlerActor.java

```diff
@@ -0,0 +1,124 @@
+        Exception e = commandRequest.getException();
+        HttpResponseStatus httpResponseStatus;
+        DssRestServiceResponse dssRestServiceResponse;
+
+        if (e instanceof DssRestRequestMappingException) {
+            context.getLog().error("Json request mapping error: ", e);
+            httpResponseStatus = HttpResponseStatus.BAD_REQUEST;
+        } else {
+            context.getLog().error("Handling rest request error: ", e);
+            httpResponseStatus = HttpResponseStatus.INTERNAL_SERVER_ERROR;
+        }
+
+        dssRestServiceResponse = findExceptionHandleMethod(commandRequest);
+
+        replyRequest(commandRequest.getRequest(), httpResponseStatus, dssRestServiceResponse);
+
+        return Behaviors.same();
+    }
+
+    private DssRestServiceResponse findExceptionHandleMethod(DssRestExceptionHandlerCommandRequest commandRequest) {
+        Map<Class<? extends Exception>, Method> serviceExceptionHandlerMap = exceptionHandlerMap.get(commandRequest.getService().getClass());
+
+        if (containMethod(serviceExceptionHandlerMap, commandRequest)){
+            Method method = serviceExceptionHandlerMap.get(commandRequest.getException().getClass());
+            return invokeMethod(method, commandRequest);
+        }
+
+        Map<Class<? extends Exception>, Method> globalExceptionHandler = exceptionHandlerMap.get(DssRestActorService.class);
+        if (containMethod(globalExceptionHandler, commandRequest)){
+            Method method = globalExceptionHandler.get(commandRequest.getException().getClass());
+            return invokeMethod(method, commandRequest);
+        }
+
+        return null;
+    }
+
+    private DssRestServiceResponse invokeMethod(Method method, DssRestExceptionHandlerCommandRequest commandRequest) {
+        try {
+            Constructor<?> declaredConstructor = method.getDeclaringClass().getDeclaredConstructor();
+            declaredConstructor.setAccessible(true);
+
+            Object exceptionHandler = declaredConstructor.newInstance();
+            Object response;
+
+            if (method.getParameterCount() == 1){
+                Method makeRequest = AbstractDssRestActorService.class.getDeclaredMethod("makeRequest", DssRestServiceActorCommandRequest.class);
+                makeRequest.setAccessible(true);
+
+                DssRestServiceRequest request = (DssRestServiceRequest) makeRequest.invoke(commandRequest.getService(), commandRequest.getRequest());
+                response = method.invoke(exceptionHandler, request.getBody());
+            } else {
+                response = method.invoke(exceptionHandler, null);
+            }
+
+            if (response instanceof DssRestServiceResponse) {
+                return (DssRestServiceResponse) response;
+            }
+        } catch (InstantiationException | InvocationTargetException | NoSuchMethodException | IllegalAccessException e) {
+            e.printStackTrace();
```

Comment:
Please use logger instead of printStackTrace

Reply:
Ok. I'll try logger.